db_test.go

```diff
@@ -1768,7 +1784,13 @@ func TestLSMOnly(t *testing.T) {
 	require.NoError(t, err)
 
 	defer db.Close()
-	require.NoError(t, db.RunValueLogGC(0.2))
+
+	vlogFiles, err := getSuffixedFiles(dir, vlogSuffix)
+	require.NoError(t, err)
+	require.Equal(t, 0, len(vlogFiles))
+	walFiles, err := getSuffixedFiles(dir, walSuffix)
+	require.NoError(t, err)
+	assert.LessOrEqual(t, 3, len(walFiles))
```

Comment:
This may be incorrect. Why should there be 3 wal files? I would expect one or two.

Maybe check if the first WAL file is ahead of the head pointer.